src/Workspaces/VisualBasic/Portable/Formatting/Engine/Trivia/VisualBasicTriviaFormatter.vb

```diff
@@ -69,7 +69,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Formatting
 
             ' line continuation
             If trivia2.Kind = SyntaxKind.LineContinuationTrivia Then
-                Return LineColumnRule.ForceSpacesOrUseAbsoluteIndentation(spacesOrIndentation:=1)
+                If trivia1.Kind = SyntaxKind.None Then
+                    Return LineColumnRule.ForceSpacesOrUseAbsoluteIndentation(spacesOrIndentation:=1)
+                ElseIf trivia1.Kind = SyntaxKind.LineContinuationTrivia Then
+                    Return LineColumnRule.ForceSpacesOrUseDefaultIndentation(spaces:=0)
+                ElseIf trivia1.Kind = SyntaxKind.commenttrivia Then
+                    Return LineColumnRule.ForceSpacesOrUseDefaultIndentation(spaces:=-1)
+                Else
+                    Return LineColumnRule.PreserveLinesWithFollowingPrecedingIndentation()
```

Comment:
can you provide comments in this code giving examples of what each of these if-blocks would correspond to and why the LineColumnRules you are returning are desirable?  For example, i have no idea why you are returning `spacesOrIndentation:=1` for `trivia1.Kind = SyntaxKind.None` or why you return `spaces:=-1` for `trivia1.Kind = SyntaxKind.commenttrivia`

Reply:
Done is newest commit